specification/network/resource-manager/readme.md

```diff
@@ -731,6 +730,9 @@ output-folder: $(go-sdk-folder)/services/network/mgmt/2018-02-01/network
 ## Suppression
 ``` yaml
 directive:
+  - suppress: RequiredPropertiesMissingInResourceModel

```

Comment:
Was this approved?

Reply:
yes, that is the same exception for ID readonly we had 1-2 months ago